build.gradle.kts

```diff
@@ -71,6 +71,8 @@ dependencies {
 
 	implementation("com.fasterxml.jackson.module:jackson-module-parameter-names:$jacksonModuleVersion")
 
+    // Temporal.io
+	implementation ("io.temporal:temporal-sdk:1.19.0")
```

Comment:
use a variable to define the dependency version (Sonar is used to raise this as an issue)